internal/application/application.go

```diff
@@ -330,13 +335,13 @@ func List(ctx context.Context, cluster *kubernetes.Cluster, namespace string) (m
 		requestctx.Logger(ctx).Error(err, "metrics not available")
 	}
 
-	// VI. load all the status of the staging jobs
+	// VI. load the stati of all staging jobs
```

Comment:
Is the plural of status "statuses", or not?

Reply:
I believe that `stati` is the plural

Reply:
Not sure if this conversation was resolved. Same for the other one :slightly_smiling_face:

Reply:
Agree what @enrichman : https://www.grammarbook.com/blog/singular-vs-plural/plural-of-status/ 
It also comes to mind more frequently

Reply:
Addressing with #2180